### PR TITLE
Bill votes

### LINKED ISSUE #300: add votes to legislation
Once https://github.com/opencivicdata/scrapers-us-municipal/issues/372 comes in, we can add vote counts to the action table for each piece of legislation

We can access these values via:

If there's a vote event and there's not any `PersonVotes` its voice vote. 

- `Bill` => `Actions` => `VoteEvent` => `PersonVote` to get list of 50 yes/no/absent values for each member
- `Bill` => `Actions` => `VoteEvent` => `VoteCount` to get summary of vote counts

`VoteEvent.motion_text` and `VoteEvent.result` are should also be displayed

Models defined here:
https://github.com/opencivicdata/python-opencivicdata/blob/master/opencivicdata/legislative/models/vote.py

### LINKED ISSUE #305: duplicate committees
Two committees are showing up twice in the list. The duplicates have identical slugs

[Aviation](https://chicago.councilmatic.org/committee/committee-on-aviation-67beb648a3f1/)
[Contracting Oversight and Equity](https://chicago.councilmatic.org/committee/committee-on-contracting-oversight-and-equity-c14ee5c77cad/)

### PR DESCRIPTION
**Pull request by derekeder**
Was able to pull the latest data from production and update the README instructions for others with access to reproduce.

This PR is a first pass at representing votes for actions related to bills. It shows a total along with the list of voters with links to their profiles. Totals are grouped by "Votes for", "Votes against" and "Other votes" which include "Absent", "Not voting" and everything else.

Here's a few examples:

Ordinance O2017-2027: Amendment of Municipal Code Title 2 by removing individuals retained as independent contractors from definition of "City employee" and "City Council employee" (failed)
Screen Shot 2023-01-03 at 4 32 25 PM

Routine committee vote on Ordinance O2011-3749 with a committee vote and a full council vote

Screen Shot 2023-01-03 at 4 39 03 PM

Screen Shot 2023-01-03 at 4 32 13 PM

There are a lot of votes that are 45+ votes in favor, which makes the list quite long. Thinking of truncating to the top 20 and adding a 'show more' button in this case

Closes #300
Closes #305 

This is branched off of `alder-attendance` so we should bring in #317 first

### CODE REVIEW

**Comment by fgregg** [comment]
i'll review code when there's a clean PR after #317.

i like this a lot. if there are no absents or nos, maybe just give indicator that vote was unanimous?

File: chicago/templates/legislation.html [hunk 1]
```diff
@@ -100,25 +100,82 @@ <h3 class="no-pad-bottom"><i class='fa fa-fw fa-list-ul'></i> History</h3>
...
             <tbody>
               {% for action in actions %}
                 <tr>
-                  <td class='nowrap text-muted small'>
+                  <td class='nowrap text-muted'>
                     <span datetime='{{action.date }}'>{{action.date}}</span>
                   </td>
-                  <td class="small">
+                  <td>
+                    {{action.organization.link_html|safe}}
```
Comment (fgregg) [inline chicago/templates/legislation.html:117]: let's use the {{url}} pattern. i'd like to deprecate the link_html stuff.

**Review by fgregg — CHANGES_REQUESTED** [review]
looks good. one change requested.

File: chicago/templates/legislation.html (hunk 1 above)
Comment (derekeder) [reply chicago/templates/legislation.html]: +1

**Comment by derekeder** [comment]
@fgregg ok this is ready to go I think. I made a few additional changes:

- using `url` instead of `link_html`
- getting distinct committees to close #305 
- moved the vote table to a sub-table inside the action cell. this helps for a simpler display when there's no voting on a bill

Here's a bill with votes:

Screen Shot 2023-01-05 at 4 50 06 PM


And one without:

Screen Shot 2023-01-05 at 4 50 52 PM

**Review by fgregg — APPROVED** [review]
lgtm!

**Pull request merged**